### PR TITLE
Completed [mavencentral mavenmetadata microbadger mozillaobservatory myget netlify] static classes into static props

### PR DESCRIPTION
**Pull request by jabbar86**

### CODE REVIEW

File: services/maven-central/maven-central.service.js [hunk 1]
```diff
@@ -52,7 +52,9 @@ module.exports = class MavenCentral extends BaseXmlService {
     },
   ]
 
-  static defaultBadgeData = { label: 'maven-central' }
+  static defaultBadgeData = {
+    label: 'maven-central',
+  }
```
Comment (calebcartwright) [inline services/maven-central/maven-central.service.js:57]: Out of curiosity, why this change?

File: services/mozilla-observatory/mozilla-observatory.service.js [hunk 2]
```diff
@@ -46,42 +46,32 @@ const documentation = `
 `
 
 module.exports = class MozillaObservatory extends BaseJsonService {
-  static get category() {
-    // TODO: Once created, change to a more appropriate category,
-    // see https://github.com/badges/shields/pull/2926#issuecomment-460777017
```
Comment (calebcartwright) [inline services/mozilla-observatory/mozilla-observatory.service.js:51]: Let's be sure that we don't drop the inline comment as part of the refactoring

File: services/maven-central/maven-central.service.js (hunk 1 above)
Comment (jabbar86) [reply services/maven-central/maven-central.service.js:57]: @calebcartwright Just to align the object format as I have done in all files.

File: services/mozilla-observatory/mozilla-observatory.service.js (hunk 2 above)
Comment (jabbar86) [reply services/mozilla-observatory/mozilla-observatory.service.js:51]: Sure will keep this in mind.Thanks

File: services/maven-central/maven-central.service.js (hunk 1 above)
Comment (calebcartwright) [reply services/maven-central/maven-central.service.js:57]: prettier will automatically wrap lines when they need to be, though generally respects the original line wrapping as well. This one is really short so it could easily fit on one line. 

No need to revert this change, but also don't feel obligated to wrap these really simple/short object literals on multiple lines either. In some cases, such as larger route definitions, they're more readable, but otherwise keeping short fields like this one line helps make the files ever so smaller :smile:

File: services/mozilla-observatory/mozilla-observatory.service.js (hunk 2 above)
Comment (calebcartwright) [reply services/mozilla-observatory/mozilla-observatory.service.js:51]: No worries. This should be good to go once you restore this comment

File: services/maven-central/maven-central.service.js (hunk 1 above)
Comment (jabbar86) [reply services/maven-central/maven-central.service.js:57]: Okay @calebcartwright  understood thank you  😄

File: services/mozilla-observatory/mozilla-observatory.service.js (hunk 2 above)
Comment (jabbar86) [reply services/mozilla-observatory/mozilla-observatory.service.js:51]: @calebcartwright changes done.

**Review by calebcartwright — APPROVED** [review]
Thanks!
